migrations/versions/5ec5c84ba61e_.py

```diff
@@ -0,0 +1,35 @@
+"""empty message
+
+Revision ID: 5ec5c84ba61e
+Revises: d1eae8b9893e
+Create Date: 2017-10-17 18:21:00.174015
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils as su
+import sqlalchemy_searchable as ss
+
+
+# revision identifiers, used by Alembic.
+revision = '5ec5c84ba61e'
+down_revision = 'd1eae8b9893e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    op.add_column('queries', sa.Column('search_vector', su.TSVectorType(),
+                                       nullable=True))
+
+    ss.sync_trigger(conn, 'queries', 'search_vector',
+                    ['name', 'description', 'query'])
```

Comment:
Will this also update all existing queries?

Reply:
Yes, looking at the [docs](https://sqlalchemy-searchable.readthedocs.io/en/latest/alembic_migrations.html#sqlalchemy_searchable.sync_trigger) it says: "Updates all rows for given search vector by running a column=column update query for given table."

I've also manually added a bunch of queries against Redash's own Postgres database in the master branch, then switch to this branch, did the Alembic upgrade and then was able to use the search feature right away.